generative/networks/blocks/selfattention.py

```diff
@@ -11,12 +11,22 @@
 
 from __future__ import annotations
 
+import importlib.util
 import math
 
 import torch
 import torch.nn as nn
 from torch.nn import functional as F
 
+if importlib.util.find_spec("xformers") is not None:
```

Comment:
Currently if the code does not find `xformers` but`use_flash_attention` is set `True` the code errors out. I think we need to self `use_flash_attention=False` in the init if `has_xformers=False`, and ideally raise a warning, too.

Reply:
Added error message in case user want to use flash attention but xformers is not installed